PYME/recipes/traits.py

```diff
@@ -10,6 +10,21 @@
 except ImportError:
     from traits.api import *
 
+# definitions below have been adapted based on https://docs.enthought.com/traits-futures/0.2/_modules/traits/trait_types.html
+# these Dict have been deprecated for a while and need to be defined here for newer traits.api modules
```

Comment:
Do we actually use any of these? Would it be better to update out code where they are used?